tools/topology/topology2/include/components/eqfir/flat.conf

```diff
@@ -0,0 +1,15 @@
+# Flat response, created with example_fir_eq.m 02-Feb-2023
+Object.Base.data."fir_eq" {
+	bytes "
```

Comment:
I'm not super keen on taking this, but we need to test and can fix this using a file reference in the future.